### ISSUE TITLE
MetaGpt not as a oneshot commander, but a continue chat backend api server.

### ISSUE THREAD

**Issue by code959437957** [geekan/MetaGPT#1365]
The most common usage case about MetaGpt is just like a one-shot command executor。

You input the task: " write me a golang hello world"

And MetaGpt will do lots of works for you, and then return you the workspace contains the code and document files。

BUT, if can we make MetaGPT as a chat backend API server。

For example, we sold shoes, we want to train a AI salesman。 

The salesman should know each shoes style, size, and it's price.

It must answer question about the shoes. provided price sheets for customers.

And if customer not willing to buy, the salesman should talk to customers, to make customer buy shoes as well.

So this is not a simple application, but a very complex requirement.

The role should have much knowledge on sales skills, and well talk ability。

MetaGPT seems can do this job, but i don't know how to make it running as a API server.

It should read and remember talk sessions。 may be keep the same customer talk history for a long time, Next time the customer came back, we should still remember what he talked。

**Comment by zhaoq1** [comment]
Yes, I think this is very important. I think the human-computer interaction mode should be optimized in the streaming interface. The current terminal interaction method is very inconvenient.

**Comment by geekan** [comment]
We will fix this issue in version 1.0
